test/libraries/PoolIdLibrary.t.sol

```diff
@@ -5,13 +5,11 @@ import {Test} from "forge-std/Test.sol";
 
 import {Currency} from "../../src/types/Currency.sol";
 import {PoolKey} from "../../src/types/PoolKey.sol";
-import {PoolId, PoolIdLibrary} from "../../src/types/PoolId.sol";
+import {PoolId} from "../../src/types/PoolId.sol";
 import {IHooks} from "../../src/interfaces/IHooks.sol";
 import {IPoolManager} from "../../src/interfaces/IPoolManager.sol";
 
 contract PoolIdLibraryTest is Test {
-    using PoolIdLibrary for PoolKey;
```

Comment:
should we create a issue to clean up on periphery/universal-router as well?

Reply:
Yeah, good idea.. I planned to clean those when updating v4-core for those repo, but better to have a issue track them